stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/extensions/internal/Channel.kt

```diff
@@ -51,13 +51,17 @@ internal fun Channel.updateLastMessage(message: Message) {
     }
 }
 
-internal fun Channel.setMember(userId: String, member: Member?) {
+internal fun Channel.setMember(userId: String, member: Member?, isUpdate: Boolean = false) {
```

Comment:
I would rather remove this default parameter and make it explicit everywhere. 
Can we change the method's name to match its behavior?
Can we also add KDoc to this method?